forge-gui/res/cardsfolder/upcoming/cyber_conversion.txt

```diff
@@ -1,6 +1,6 @@
 Name:Cyber Conversion
 ManaCost:U U
 Types:Instant
-A:SP$ SetState | ValidTgts$ Creature | Mode$ TurnFace | FaceDownSetType$ Artifact & Creature & Cyberman | FaceDownPower$ 2 | FaceDownToughness$ 2 | SpellDescription$ Turn target creature face down. It's a 2/2 Cyberman artifact creature.
+A:SP$ SetState | ValidTgts$ Creature | Mode$ TurnFaceDown | FaceDownSetType$ Artifact & Creature & Cyberman | FaceDownPower$ 2 | FaceDownToughness$ 2 | SpellDescription$ Turn target creature face down. It's a 2/2 Cyberman artifact creature.
```

Comment:
@Agetian does this maybe need AI logic to not target an already face down card? or is that already handled in SetStateAI ?

Reply:
added sanity checks